firmware/application/apps/acars_app.cpp

```diff
@@ -98,9 +98,10 @@ ACARSAppView::ACARSAppView(NavigationView& nav) {
 		logging = v;
 	};
 	
+	make_new_directory(LOG_ROOT_DIR);
 	logger = std::make_unique<ACARSLogger>();
 	if (logger)
-		logger->append("acars.txt");
+		logger->append( LOG_ROOT_DIR "/ACARS.TXT" );
```

Comment:
Would it be easier to change this API to take two args:
1) The folder path
2) The file name

That way you could consolidate the make_new_directory calls and just rely on the logger to ensure the path exists?
That would make it easier to support subfolders too (like LOGS/POCSASG).

Reply:
True

Reply:
For me that would be in another PR because after looking at it the changes are not as simple as the ones here.